test/test_retry.py

```diff
@@ -400,7 +399,7 @@ def test_respect_retry_after_header_sleep(
     ) -> None:
         retry = Retry(respect_retry_after_header=respect_retry_after_header)
 
-        with freezegun.freeze_time("2019-06-03 11:00:00", tz_offset=0), mock.patch(
+        with mock.patch("time.time", return_value=1559559600.0), mock.patch(
```

Comment:
Is there a way to start from something more readable than 1559559600? What do you think about `datetime.fromisoformat("2019-06-03 11:00:00Z").timestamp()`?

Reply:
how about constructing a datetime.datetime directly?